composer.json

```diff
@@ -23,11 +23,11 @@
   "prefer-stable": true,
   "license": "GPL-2.0-or-later",
   "require": {
-    "php": ">=5.6.0",
+    "php": ">7.3",
     "composer/composer": "^1.4",
     "consolidation/robo": "^1.0",
     "symfony/filesystem": "^3.2",
-    "drush/drush": "^8.1"
+    "drush/drush": "~10.0"
```

Comment:
I am fine with only drush 10, or would it work with 9 also?

Reply:
I think Drupal 9 does not have a purpose anymore, but sure all would/should work with 9 as well, we could go with 9||10

Reply:
So, this works with Drupal 9 as well

https://travis-ci.com/github/BurdaMagazinOrg/thunder-web/builds/173992346#L1481